src/__tests__/test.spec.ts

```diff
@@ -14,7 +12,7 @@ const standardSchema = new Schema({
 })
 const listSchema = new Schema({
    nodes: addListNodes(standardSchema.spec.nodes, "paragraph block*", "block"),
-   marks: standardSchema.markSpec
+   marks: standardSchema.spec.marks
```

Comment:
That's a good question! I'll look into that.

Reply:
Oh wow, so this field was removed in 2017 https://github.com/ProseMirror/prosemirror-model/commit/a337a4af5901d56878a0175c92524cc227216473, so this has been undefined forever.

It wasn't causing an error because none of the list schema test cases used marks, so the serializer wasn't actually trying to access the schema's marks.

I'll add a test case to make sure it'll fail in the future if the mark spec is ever undefined.

Reply:
Thanks for hunting that down! Wow—glad that got caught now! Was it TypeScript that actually noticed it was undefined?